### ISSUE TITLE
The game pop up createdxgifactory2 error when i launch battlefront 2

### ISSUE THREAD

**Issue by mfon223** [SleepKiller/shaderpatch#111]
when i launch the game, it pop out this error, do u know how to fix this??
![Capture](https://user-images.githubusercontent.com/66712674/84227373-d76b8700-ab16-11ea-9915-8450988ece62.PNG)

**Comment by SleepKiller** [comment]
This is typically in indicator that your PC doesn't meet the Windows 10, 1809 requirement listed on the Release/Download page. Regretfully I have to say that there unfortunately isn't a way to fix this, aside from getting Windows 10, but that is possibly undesirable/untenable for you.

**Comment by mfon223** [comment]
i have installed window 10 pro a few day ago, but i revert back to window 7 because after i installing window 10 my game fps drop example minecraft, but i tried to install shader patch battlefront 2 when i am still using window 10, but when i launch the game, it suddenly closed by itself

**Comment by mfon223** [comment]
> This is typically in indicator that your PC doesn't meet the Windows 10, 1809 requirement listed on the Release/Download page. Regretfully I have to say that there unfortunately isn't a way to fix this, aside from getting Windows 10, but that is possibly undesirable/untenable for you.

**Comment by mfon223** [comment]
> This is typically in indicator that your PC doesn't meet the Windows 10, 1809 requirement listed on the Release/Download page. Regretfully I have to say that there unfortunately isn't a way to fix this, aside from getting Windows 10, but that is possibly undesirable/untenable for you.

i have installed window 10 a few day ago, but i revert it back to window 7, and when i am still using window 10, when i launch the game, it just closed by itself

**Comment by SleepKiller** [comment]
Very annoying. Shader Patch should (hopefully) have created a file named `shader patch.log` in the same directory as the game. If you could post it's contents here that'd be great, it might contain some clues as to why it's crashing.

**Comment by mfon223** [comment]
can this run window 10 and battlefront 2 shader patch ?
![Capture](https://user-images.githubusercontent.com/66712674/84241069-86b55780-ab31-11ea-896b-efaf5b41dc0a.PNG)

**Comment by mfon223** [comment]
?

> Very annoying. Shader Patch should (hopefully) have created a file named `shader patch.log` in the same directory as the game. If you could post it's contents here that'd be great, it might contain some clues as to why it's crashing.

can this run window 10 and battlefront 2 shader patch ?
![Capture](https://user-images.githubusercontent.com/66712674/84241180-b7958c80-ab31-11ea-9cb8-3a5528867340.PNG)

**Comment by SleepKiller** [comment]
I'm not 100% sure but I don't think the iGPU in the i5-2540M meets the Direct3D Feature Level requirement. Shader Patch requires Feature Level 11_0 and I think the iGPU in that CPU only goes up to Feature Level 10_1. 

I could be wrong, I had a hard time finding that information on Intel's website but I think the answer is no it can't run the v1.0 and later releases. It should be able to run the old v0.9 releases (which where before the D3D11 switch) if that interests you https://github.com/SleepKiller/shaderpatch/releases/tag/v0.9.1

**Comment by mfon223** [comment]
I have already install shader patch v0.9.1, but when i enable config, it didn't change anything, bloom effects still didn't change except the droideka shield effects have reflection effects, how to fix this??

**Comment by mfon223** [comment]
> I'm not 100% sure but I don't think the iGPU in the i5-2540M meets the Direct3D Feature Level requirement. Shader Patch requires Feature Level 11_0 and I think the iGPU in that CPU only goes up to Feature Level 10_1.
> 
> I could be wrong, I had a hard time finding that information on Intel's website but I think the answer is no it can't run the v1.0 and later releases. It should be able to run the old v0.9 releases (which where before the D3D11 switch) if that interests you https://github.com/SleepKiller/shaderpatch/releases/tag/v0.9.1

I have already install shader patch v0.9.1, but when i enable config, it didn't change anything, bloom effects still didn't change except the droideka shield effects have reflection effects, how to fix this??

**Comment by SleepKiller** [comment]
Damn it, I started writing out a reply to this, got distracted and forgot about it for a week. Sorry about that!

Anyway, I assume you're referring to the enabling the Effects System? If so there are two key things to know. 

The first is that ingame bloom must be enabled for it to work, it's what it uses to tell when it's time to apply post processing.

The second is that it has to be explicitly enabled by the user or a mod. In v0.9 this is done on the Developer Screen which you bring up by hitting `\` then looking for and checking `Enable` somewhere under the `Effects` window. The ability for a end user (not just a mod/map) to specify a persistent config was added in v1.0, so unfortunately this process must be repeated each time you start the game.
